### PR TITLE
vfp: Get rid of warnings

### PR DESCRIPTION
**Pull request by lioncash**
Silences some warnings when compiling with VS.

### CODE REVIEW

File: src/core/arm/skyeye_common/vfp/vfp_helper.h [hunk 1]
```diff
@@ -144,8 +144,8 @@ static inline void mul64to128(u64* resh, u64* resl, u64 n, u64 m)
     u32 nh, nl, mh, ml;
     u64 rh, rma, rmb, rl;
 
-    nl = n;
-    ml = m;
+    nl = static_cast<u32>(n & 0xFFFFFFFF);
+    ml = static_cast<u32>(m & 0xFFFFFFFF);
```
Comment (yuriks) [inline src/core/arm/skyeye_common/vfp/vfp_helper.h]: Why the ands?

**Comment by lioncash** [comment]
Those were added just to make intent explicit. I can remove them if they aren't wanted.

File: src/core/arm/skyeye_common/vfp/vfp_helper.h (hunk 1 above)
Comment (lioncash) [inline src/core/arm/skyeye_common/vfp/vfp_helper.h]: Those were added just to make intent explicit. I can remove them if they aren't wanted.

Comment (yuriks) [inline src/core/arm/skyeye_common/vfp/vfp_helper.h]: With both the cast and the and, it seemed to me that the and was supposed to be achieving something else, so I was a bit perplexed by it for a bit. It's fine either way to me.

**Comment by bunnei** [comment]
LGTM

**Pull request merged**